lib/internal/constants.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  // Alphabet chars.
+  CHAR_UPPERCASE_A: 65, /*A*/
```

Comment:
nit: for readability it might help to change these comments to `//` style

Reply:
@devdazed what about comment like this: `// /`. Not it's `/*/*/`?

Reply:
this is probably more subjective than i realized, i'll leave it up to your judgement 👍

Reply:
I would at least put spaces and quotes around them like `/* 'A' */` which also makes the original suggestion less crazy since it would be `// '/'` which is at least a little clearer.